torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -341,11 +339,6 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
         updated_val = at::unsqueeze(updated_val, axis);
       }
       return c10::optional<at::Tensor>(updated_val);
-    } else {
-      TORCH_WARN(
-          "Constant folding - unsupported opset version. "
-          "Constant folding not applied.");
-      return c10::nullopt;
```

Comment:
Is it ok to remove the return statement too?

Reply:
I will change the else if to else.

Reply:
I suppose the early return logic still needs to be kept?

Reply:
Interesting. If we are observing spammy constant folding warning, that must imply the guarded else branch of `opset < 9` is hit, right? Another bug somewhere else?

Reply:
Nooooooo